airflow/www/views.py

```diff
@@ -93,7 +93,7 @@ def get_date_time_num_runs_dag_runs_form_data(request, session, dag):
     if dttm:
         dttm = timezone.parse(dttm)
     else:
-        dttm = dag.latest_execution_date or timezone.utcnow()
+        dttm = dag.get_latest_execution_date() or timezone.utcnow()
```

Comment:
```suggestion
        dttm = dag.get_latest_execution_date(session) or timezone.utcnow()
```

Reply:
There are more of it where we can just pass the session which is better than recreating it, don't you think?